src/autocomplete.js

```diff
@@ -420,25 +431,8 @@ class Autocomplete extends Component {
     // because the items are rerendered every time we call the children
     // we clear this out each render and
     this.items = []
-    const {
-      children,
-      // eslint-disable-next-line no-unused-vars
-      defaultSelectedItem,
-      // eslint-disable-next-line no-unused-vars
-      getValue,
-      // eslint-disable-next-line no-unused-vars
-      getA11yStatusMessage,
-      // eslint-disable-next-line no-unused-vars
-      defaultHighlightedIndex,
-      // eslint-disable-next-line no-unused-vars
-      onChange,
-      ...rest
-    } = this.props
-    return (
-      <div {...rest} ref={this.rootRef} onClick={this.handleClick}>
-        {children(this.getControllerStateAndHelpers())}
-      </div>
-    )
+    const {children} = this.props
+    return children(this.getControllerStateAndHelpers())
```

Comment:
I generally agree with you, but I think that `getRootProps` will only be necessary in some use cases, and I'd prefer to not require folks to use it :smile: